get_deps.sh

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env bash
+set -e
 
 BASE_DIRECTORY=`pwd`
 
-DEPS_DIRECTORY=${BASE_DIRECTORY}/deps
+# Allow different deps for different platforms:
+PLATNAME=${OSTYPE}
+if [ -e /etc/debian-version ]; then
+    PLATNAME=${PLATNAME}-deb
+fi
+
+DEPS_DIRECTORY=${BASE_DIRECTORY}/deps-${PLATNAME}
```

Comment:
It's good to have platform specific `deps`, but you'll need to make sure the `DEPS_PATH` is set appropriately in the `Makefile`, see https://github.com/RedisAI/RedisAI/blob/master/src/Makefile#L8
(this is the cause of the failing test - it's not finding `redismodule.h` because it's looking for it in `deps`)